config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :merchants do
+    resources :items, controller: 'merchant_items'
+  end
+  get 'merchants/:merchant_id/dashboard', to: 'merchants#dashboard'
+
+  get '/merchants/:merchant_id/invoices',to: 'merchant_invoices#index'
```

Comment:
Looks good but we will have to check frequently if these need to be updated or added to.